langchain/vectorstores/chroma.ts

```diff
@@ -0,0 +1,194 @@
+import fs from "fs/promises";
+import path from "path";
+import type {
+  ChromaClient as ChromaClientT,
+} from "chromadb";
+import { ChromaClient as ChromaClientO } from "chromadb";
+
+import { Embeddings } from "../embeddings/base";
+
+import { DocStore, SaveableVectorStore } from "./base";
+import { Document } from "../document";
+
+let ChromaClient: typeof ChromaClientT | null = null;
+
+try {
+  // eslint-disable-next-line global-require,import/no-extraneous-dependencies
+  ({ ChromaClient } = require("chromadb"));
+} catch {
+  // ignore error
+}
+ChromaClient = ChromaClientO;
+
+export interface ChromaLibArgs {
+  space: string;
+  numDimensions?: number;
+  collection_name: string,
```

Comment:
Not too particular about this, but do we want to make all the args the same case?

Reply:
ah! should not be snakecase. thank you